lib/presentation/features/details/bloc/details_state.dart

```diff
@@ -8,7 +8,7 @@ class DetailsState extends Equatable {
   final bool loading;
   final ViewDataCharacterDetails? characterDetails;
   final List<ViewDataSeries>? series;
-  final String? error;
+  final bool? error;
```

Comment:
Now it is true/false so you no need to have it nullable

Reply:
done